olive/passes/onnx/perf_tuning.py

```diff
@@ -441,7 +441,9 @@ def get_benchmark(
 
             start_time = time.perf_counter()
             evaluator = OliveEvaluatorFactory.create_evaluator_for_model(model)
-            metric_result = evaluator.evaluate(model, self.data_root, [latency_metric], self.config.device, None)
+            metric_result = evaluator.evaluate(
+                model, self.data_root, [latency_metric], self.config.device, inference_settings["execution_provider"]
```

Comment:
IIRC, the inference_settings will be packed into `latency_metric`. So I think you may need to revert the changes in this line once you decide to pulish this PR.